### ISSUE TITLE
(regression) 0.9.x breaking moves for custom actor types

### ISSUE THREAD

**Issue by asacolips** [asacolips-projects/pbta#107]
In GitLab by @GraithTiger on Feb 19, 2024, 22:24

Moves within custom actor types appear to be broken upon migrating from 0.8.1 to 0.9.x.
* Upon migration, all moves within the custom actor parent are deleted, except the first move in the first listed category, which is duplicated twice.
* The move edit dialog does not pop up, generating an error: `TypeError: An error occurred while rendering PbtaItemSheet 44. validCharacterType[(intermediate value)] is undefined`
* Moves are unable to be dragged-and-dropped into the custom actor from either the items tab or the compendium. Dragging moves between categories does work, however.

**Comment by asacolips** [comment]
In GitLab by @mclemente on Feb 20, 2024, 22:44

Could you provide your TOML and export the data of an affected actor so I can test it?

**Comment by asacolips** [comment]
In GitLab by @GraithTiger on Feb 21, 2024, 19:58

Here's the requested data. It looks like the `baseType` of the actor isn't being respected. It's rendering like a character instead of a NPC.

Sheet config:
[pbta_config.toml](https://gitlab.com/asacolips-projects/foundry-mods/pbta/uploads/57435f2c33db5b4bba8454a6afecc558/pbta_config.toml)

[Actor pre-migration](https://gitlab.com/asacolips-projects/foundry-mods/pbta/uploads/6f12648463db0330599ad0ea6d67295b/fvtt-Actor-0278-wingull-Xc9yDp4sNQZoco7p.json)

[Actor post-migration](https://gitlab.com/asacolips-projects/foundry-mods/pbta/uploads/35934a78fe89eaf2ba19f2bfe19ba174/fvtt-Actor-0278-wingull-Xc9yDp4sNQZoco7p_2.json)

**Comment by asacolips** [comment]
In GitLab by @Asacolips on Feb 22, 2024, 13:51

mentioned in commit 594a0bf39ef4c7a73346ed38a871db31c322e522

**Comment by asacolips** [comment]
In GitLab by @mclemente on Feb 22, 2024, 13:54

Thank you. I've made a typo that makes the system assume all Other-type actors' base types to be "character" instead.

The good news is that items weren't deleted, they just weren't being read by the sheet since it was assuming it was a character.

I've fixed the Move editing and couldn't reproduce the drag-and-drop when I got to it, so they probably were related.
